CurrencyConverter/CurrencyConverter/ViewController.swift

```diff
@@ -23,17 +28,42 @@ class ViewController: UIViewController {
     // MARK: - Actions
     
     @IBAction func convertButtonTapped(_ sender: UIButton) {
+        guard let text = fromCurrencyTextField.text,
+            let dollars = Double(text) else {return}
+        let convertedCurrency = convert(dollars)
+        toCurrencyTextField.text = "\(convertedCurrency)"
         
     }
     
     @IBAction func cadButtonTapped(_ sender: UIButton) {
-        
+        sender.isSelected.toggle()
+        mxnButton.isSelected.toggle()
```

Comment:
I like that you used the toggle statements. While it functions properly it what is actually happening can be a bit confusing when just reading the code. I might suggest adding a comment about of the toggle statements just to make it a bit more understandable to anyone reading it.